### PR TITLE
Travis CI: Test on current Node.js releases 10, 12, 14

### PR DESCRIPTION
**Pull request by cclauss**
https://github.com/nodejs/Release

FYI @zamoraignacio

### CODE REVIEW

**Comment by cclauss** [comment]
`NODE_GYP_FORCE_PYTHON=python2 npm install ffi` seems to work on Ubuntu on Node.js 10 if legacy Python is properly installed.  Node.js 12 and 14 do not seem to work.

**Comment by huan** [comment]
I'd like to see this to be merged and looking forward to using ffi with Node.js v14!

**Comment by cclauss** [comment]
Anyone can review any pull request on GitHub.
If you want to give project maintainers confidence that a pull request is useful for the project:
1. Click the ` Files changed` tab above
2. Click the `Review changes` button
3. Leave a comment
4. Click the `Approve` radio button
5. Click the `Submit review` button

Now project maintainers know that you have taken the time to study the changes made and believe that they improve the codebase.

File: .travis.yml [hunk 1]
```diff
@@ -10,19 +9,18 @@ addons:
...
+    - TRAVIS_NODE_VERSION="14"
 
 install:
   - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
   - if [[ $TRAVIS_OS_NAME == "linux" ]]; then export CXX=g++-4.8; fi
   - $CXX --version
   - PATH="`npm bin`:`npm bin -g`:$PATH"
   # Install dependencies and build
-  - npm install
+  - NODE_GYP_FORCE_PYTHON=python2 npm install
```
Comment (cclauss) [inline .travis.yml:23]: Do we really need legacy Python?!?  It died over a year ago and should now be considered an insecure platform.

File: appveyor.yml [hunk 2]
```diff
@@ -2,32 +2,19 @@
...
-  - ps: if($env:nodejs_version -ne "0.8") {Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version)}
-  # Node 0.8 comes with a too obsolete npm
-  - IF %nodejs_version% == 0.8 (npm install -g npm@1.4.28)
-  # Install latest NPM only for node.js versions until built in node-gyp adds io.js support
-  # Update is required for node.js 0.8 because built in npm(node-gyp) does not know VS2013
-  - IF %nodejs_version% LSS 1 (npm install -g npm@2)
-  - IF %nodejs_version% LSS 1 set PATH=%APPDATA%\npm;%PATH%
-  # Typical npm stuff.
-  - npm install --msvs_version=%MSVS_VERSION%
+  - npm install --python="C:\Python27"  # "C:\Python27-x64"
```
Comment (cclauss) [inline appveyor.yml:17]: Do we really need legacy Python?!?  It died over a year ago and should now be considered an insecure platform.

**Review by cclauss — COMMENTED** [review]
The current version of Python is v3.9.1

**Comment by vincent-bo** [comment]
When do you think we will have an ffi version for node.js v14 ?

Thanks

**Comment by yegorich** [comment]
@vincent-bouffigny have you already looked at this project? https://github.com/node-ffi-napi

**Comment by vincent-bo** [comment]
yes but there are installation problems with node> 10 ...

**Pull request closed without merging**